src/test/resources/pages/adminActivityLogPage.html

```diff
@@ -261,9 +256,11 @@
             <ul class="nav navbar-nav pull-right">
                 <li>                
                     <a href="/logout.jsp" class="nav logout">
-                        <span class="glyphicon glyphicon-user"></span>
-                        Logout  (<span title="" data-placement="bottom" data-toggle="tooltip" class="text-info" data-original-title="defaultAdmin"> defaultAdmin
-                        </span>) 
+                        <span class="glyphicon glyphicon-user"></span> Logout
+                        
+                        (<span title="" data-placement="bottom" data-toggle="tooltip" class="text-info" data-original-title="defaultAdmin">
+                                defaultAdmin
```

Comment:
About the reason the Admin tests sometimes fail, it is because of the login name. It could be the account name from other test when we run them concurrently. Probably putting a wildcard for it will solve the problem.